### ISSUE TITLE
[Queue Listener] Scheduled tasks don't finish + First task in schedule repeats three times

### ISSUE THREAD

**Issue by TheFlash787** [pterodactyl/panel#1272]
* Panel or Daemon: Panel
* Version of Panel/Daemon: 
* Server's OS: Ubuntu 16.04
* Your Computer's OS & Browser: 

**Describe the bug**

 If I have a schedule of lets say 4 tasks, when the time comes, It does the first one and repeats it 3 times. It then stops the schedule and acts like it's completed (Then repeats at the next interval). These schedules worked before the update. I've tried a reinstall of the queue listener, but no success. https://files.modrealms.net/screenshots/16.21.52-07.08.18.png One of the tasks seemed to have failed in the logs but the others seem fine: https://hastebin.com/omovisepap.java (Queue listener log)

Panel: **Panel Logs:** https://bin.ptdl.co/p4qmv/ **PHP Version:** PHP 7.2.7
**To Reproduce**
1. Create a schedule with any command (with more tasks preferably)
2. Wait for the time to pass until the schedule begins
3. Notice that the first task in the schedule runs 3 times and doesn't progress
4. Schedule shows as completed and repeats at the next interval

**Expected behavior**
Each task in the schedule should run one after another and complete when finished. 

Thanks!

**Comment by DaneEveritt** [comment]
It honestly just looks like your queue worker got clogged up and had a backlog that it ran on you, but I also wouldn't be surprised if this is actually broken.

It also appears whatever that last queued command was failed three times in a row, so the whole sequence was stopped, which would explain your confusion there.

Is this a consistent problem? Can you provide logs that only contain one run through of the scheduled task, its really hard to follow what is and is not related to that one task.

**Comment by TheFlash787** [comment]
Hey!

Yes this is consistent. Here is a log of just one schedule with only one task needed: https://hastebin.com/ofeqibepoh.java

It seems to process it 3 times and then fail the lot. Which is odd. This happens to all of the tasks that I have running for each server.

Thanks!

**Comment by xion87** [comment]
I noticed it too, also the crash detector still don't work, i found my server-game-bungee still offline  after the restart

**Comment by fabri2000779** [comment]
i can confirm this, in our production enviroment we had many reports about the task spaming three times the command, i thought that was just one issue of our end and nothing related to ptero, it is courius because this start happening after the update.

**Comment by TheFlash787** [comment]
I have found out that this issue also seems to occur only when running 0.6.X. I've downgraded to 0.5.6 and the issue seems to disappear. This issue will remain open of course so that it can be fixed in another 0.6.(4?) build. Thanks!

**Comment by TheFlash787** [comment]
Very odd then.. Perhaps if others here (who are having the same issue) also downgrade, we can see if was definitely (in some way) caused by the update. 

Thanks!

**Comment by xion87** [comment]
yeah i fixed it by downgrading

**Comment by fabri2000779** [comment]
@TheFlash787 i imagine that for the downgrade i need to rebuild all the servers and restart all the servers aswell correct?

**Comment by DaneEveritt** [comment]
@TheFlash787 can you grab the exception from the `failed_jobs` table on that triple run? Its running three times because it fails three times (at least the panel thinks it is failing).

I suppose it would make more sense in that case that the command sending is returning the wrong status code to the Panel which in turn thinks it failed so it tries again (even though it worked fine).

**Comment by fabri2000779** [comment]
also @DaneEveritt here is the log of the table that you requested:

https://hastebin.com/ubufanudeh.sql

**Comment by DaneEveritt** [comment]
@fabri2000779 what version of the Daemon is this?

**Comment by fabri2000779** [comment]
6.3 the latest one

**Comment by TheFlash787** [comment]
@DaneEveritt Yeah, that's the log you need ^

Thank you @fabri2000779 I hope that helps Dane. (6.0.3)

**Comment by TheFlash787** [comment]
Infact, in the `failed_jobs` table, I have a slightly different exception: https://hastebin.com/yiqokozeca.java

**Comment by xion87** [comment]
me https://gist.github.com/xion87/3b56fc173573fb4561fd60c779d64582

**Comment by fabri2000779** [comment]
this case maybe need to be re-open. the issue persist on the latest build 6.4 the failed job table has the same log we already tried re-building the servers. Also this time instead of threee time of the command repeat it is twice and with a delay of 30 minutes.

Another good issue is that when a server crash the daemon do not auto restart it, error what the console shows: 
Startup script './start.sh' does not exist! Stopping server.
> [09:26:33 INFO]: Stopping server
[Pterodactyl Daemon] Server marked as OFF

let me know if you requere of any log to provide it here.

**Comment by parkervcp** [comment]
For the second issue it's unrelated to this and needs to be it's own issue all together.
